src/core_plugins/kbn_vislib_vis_types/public/controls/gauge_options.html

```diff
@@ -0,0 +1,282 @@
+<div>
+  <div class="kuiSideBarFormRow">
+    <label class="kuiSideBarFormRow__label" for="gaugeType">
+      Gauge Type
+    </label>
+    <div class="kuiSideBarFormRow__control">
+      <select
+        id="gaugeType"
+        class="kuiSelect kuiSideBarSelect"
+        ng-model="vis.params.gauge.gaugeType"
+        ng-options="mode for mode in vis.type.params.gaugeTypes"
+      ></select>
+    </div>
+  </div>
+
+  <div class="kuiSideBarFormRow">
+    <label class="kuiSideBarFormRow__label" for="percentageMode">
+      Percentage Mode
+    </label>
+    <div class="kuiSideBarFormRow__control">
+      <input class="kuiCheckBox" id="percentageMode" type="checkbox" ng-model="vis.params.gauge.percentageMode">
+    </div>
+  </div>
+
+  <div class="kuiSideBarFormRow">
+    <label class="kuiSideBarFormRow__label" for="verticalSplit">
+      Vertical Split
+    </label>
+    <div class="kuiSideBarFormRow__control">
+      <input class="kuiCheckBox" id="verticalSplit" type="checkbox" ng-model="vis.params.gauge.verticalSplit">
+    </div>
+  </div>
+
+  <div class="kuiSideBarFormRow">
+    <label class="kuiSideBarFormRow__label" for="showLabels">
+      Show Labels
+    </label>
+    <div class="kuiSideBarFormRow__control">
+      <input class="kuiCheckBox" id="showLabels" type="checkbox" ng-model="vis.params.gauge.labels.show">
+    </div>
+  </div>
+
+  <div class="kuiSideBarFormRow" ng-show="vis.params.gauge.labels.show">
+    <label class="kuiSideBarFormRow__label" for="subText">
+      Sub Text
+    </label>
+    <div class="kuiSideBarFormRow__control">
+      <input
+        id="subText"
+        class="kuiInput kuiSideBarInput"
+        ng-model="vis.params.gauge.style.subText"
+        type="text"
+      >
+    </div>
+  </div>
+
+  <div class="kuiSideBarFormRow" ng-hide="vis.params.gauge.type === 'simple'">
+    <label class="kuiSideBarFormRow__label" for="extendRange">
+      Auto Extend Range
```

Comment:
consider adding little`kbn-info` tooltip with short explanation. It isn't straightforward.